src/type/index.ts

```diff
@@ -46,7 +46,6 @@ export * from './exclude/index'
 export * from './extends/index'
 export * from './extract/index'
 export * from './function/index'
-export * from './guard/index'
```

Comment:
May want to add this back as `../guard/type/index` so this isn't a breaking change.